src/components/Application/PageElements.tsx

```diff
@@ -86,13 +95,88 @@ const PageElements: React.FC<PageElementProps> = ({
   if (isReview)
     return (
       <Form>
-        {elements.map((element) => {
-          return <p>Review Element</p>
-        })}
+        <Segment.Group>
+          {elements.map(({ element, thisReviewLatestResponse }) => (
+            <Segment key={`question_${element.id}`}>
+              <Grid columns="equal">
+                <Grid.Column floated="left">
+                  <SummaryViewWrapperNEW {...getSummaryViewProps(element)} />
+                </Grid.Column>
+                <Grid.Column floated="right" textAlign="right">
+                  <ReviewButton
+                    reviewResponse={thisReviewLatestResponse as ReviewResponse}
+                    summaryViewProps={getSummaryViewProps(element)}
+                  />
+                </Grid.Column>
+              </Grid>
+              <ReviewResponseComponent
+                reviewResponse={thisReviewLatestResponse as ReviewResponse}
+                summaryViewProps={getSummaryViewProps(element)}
+              />
+            </Segment>
```

Comment:
The approve is looking good
<img width="714" alt="Screen Shot 2021-03-06 at 5 15 43 PM" src="https://user-images.githubusercontent.com/16461988/110194975-6b009700-7ea0-11eb-95db-1154f0a6cce0.png">

But I think the Decline need another line with the comment?
<img width="725" alt="Screen Shot 2021-03-06 at 5 16 11 PM" src="https://user-images.githubusercontent.com/16461988/110194978-6fc54b00-7ea0-11eb-9ab3-2bd10a3bbe0a.png">

Reply:
3c4f0d27981fe01c5c961b5d2d51c44b2c1b5e5c

Sorry not making a huge effort on these styles atm